src/components/grid.js

```diff
@@ -163,3 +163,9 @@ export const Row = props => (
 export const Col = props => (
   <Flex block px={[1, 3, 60, 42, 48, 48]} {...props} />
 )
+
+export const CompensatedRow = props => <Flex mx={[-1, -1.5, -2]} {...props} />
+
+export const CompensatedCol = props => (
+  <Flex block px={[1, 1.5, 2]} {...props} />
```

Comment:
I don't think is a problem created with the newly added code. I've mentioned this 'double-gutter' problem with the current `Col` and `Row` implementation.

The case you're referring, the `TitleAndBody` uses regular `Col` component which indeed has that problem. This `Compensated` components were created exactly to address this issue..
When openning this MR I wrote a couple of comments, and one was:

> Created compensated cols to better adapt to current design specs (gutter width)
> 
> Our design specs show our gutter width at around 48px but our current Col components use a 48px each, which causes our grid is visually the double of what it needs (48+48).
> I don't know if this is a known issue or not so until we can establish a convention, I created the CompensatedCol and CompensatedRow (probably not the best names, but I'm hoping they're temporary) that allow for the single gutter width."

Reply:
Yeah you're right. The existing `Col` creates this problem too. But I don't think the new components fix the issue.

It still uses `px` - i.e. padding on both sides. So whenever you have columns next to each other, there will still be padding on the right of one & padding on the left of the one next to it...

Am I missing something?

Reply:
Yes, there's padding on the left and right of two adjacent cols, but that padding is half the gutter, so when summed up, the gutter should stay visually correct.
Does that makes sense?

Reply:
I'm closing this as per we discussed on our slack call. This may turn into the 'default' col solution, let's just get a GO from the design team for fixing the rest of the 'double-gutter' situations